src/Domain.LinnApps/Domain.LinnApps.csproj

```diff
@@ -16,5 +16,6 @@
 		<PackageReference Include="Linn.Common.Persistence" Version="4.2.0" />
 		<PackageReference Include="linn.common.rendering" Version="1.0.0" />
 		<PackageReference Include="Linn.Common.Reporting" Version="1.9.0" />
+		<PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.13" />
```

Comment:
There is probably a better way forward than this the more I think about it ...I am wondering if we could just have our repository FilterBy() methods force the query to be executed inside them i.e. await ToListAsync() (so build up all of your filtering and ordering expressions _before_ you pass some expression to the FilterBy() by which point you are acknowleding that the query is ready to evaluate and no further filtering will occur)... This would keep the domain pure / remove the responsibility of deciding **when** to actually execute the sql query from the domain which is arguably outside of its wheelhouse anyway. But yeah that will require some Linn.Common changes which I can take a look at soon so feel free to leave this here for now